app/src/main/java/com/nextcloud/talk/activities/CallActivity.java

```diff
@@ -2701,7 +2797,7 @@ public void onComplete() {
 
         /**
          * Adds the local participant nick to offers and answers.
-         *
+         * <p>
```

Comment:
is never closed

Reply:
actually <p> seems to be added by autoformat from now on! But it's intended to not close it, see https://metaterasology.github.io/docs/workflow/documentationJavaDoc.html#html-tags